packages/shared/client/ReplayClient.ts

```diff
@@ -244,6 +249,11 @@ export class ReplayClient implements ReplayClientInterface {
     return sessionId;
   }
 
+  releaseSession(): void {
+    client.Recording.releaseSession({ sessionId: this.getSessionIdThrows() });
+    resetSession();
+  }
```

Comment:
needed to test multiple recordings. not sure if this is the best approach or not